google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/RemoveFeedItemAttributeValue.java

```diff
@@ -0,0 +1,261 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.ads.googleads.examples.remarketing;
+
+import com.beust.jcommander.Parameter;
+import com.google.ads.googleads.examples.utils.ArgumentNames;
+import com.google.ads.googleads.examples.utils.CodeSampleParams;
+import com.google.ads.googleads.lib.GoogleAdsClient;
+import com.google.ads.googleads.lib.utils.FieldMasks;
+import com.google.ads.googleads.v2.enums.FlightPlaceholderFieldEnum.FlightPlaceholderField;
+import com.google.ads.googleads.v2.errors.GoogleAdsError;
+import com.google.ads.googleads.v2.errors.GoogleAdsException;
+import com.google.ads.googleads.v2.resources.FeedAttribute;
+import com.google.ads.googleads.v2.resources.FeedItem;
+import com.google.ads.googleads.v2.resources.FeedItemAttributeValue;
+import com.google.ads.googleads.v2.services.FeedItemOperation;
+import com.google.ads.googleads.v2.services.FeedItemServiceClient;
+import com.google.ads.googleads.v2.services.GoogleAdsServiceClient;
+import com.google.ads.googleads.v2.services.GoogleAdsServiceClient.SearchPagedResponse;
+import com.google.ads.googleads.v2.services.MutateFeedItemResult;
+import com.google.ads.googleads.v2.services.MutateFeedItemsResponse;
+import com.google.ads.googleads.v2.services.SearchGoogleAdsRequest;
+import com.google.ads.googleads.v2.utils.ResourceNames;
+import com.google.common.collect.ImmutableList;
+import com.google.protobuf.Int64Value;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Map;
+
+/**
+ * Removes a the FLIGHT_SALE_PRICE FeedItemAttributeValue in a flights feed. To create a flights
+ * feed, run the AddFlightsFeed.java example. This example is specific to feeds of type
+ * DYNAMIC_FLIGHT, and specifically removes the FLIGHT_SALE_PRICE attribute, so the
```

Comment:
Instead of making this only work for FLIGHT_SALE_PRICE, can you make the FlightPlaceholderField enum value a command-line parameter?